mgmt/mgmt-core/src/main/java/deployment/mgmt/process/start/StartCommandImpl.java

```diff
@@ -23,15 +23,52 @@
 public class StartCommandImpl implements StartCommand {
     private final PropertyService propertyService;
     private final PreStartStep preStartStep;
+    private final PostStartStep postStartStep;
     private final StartStrategy startStrategy;
 
     @Override
     public void startWithArgs(String[] services, String... args) {
         announce("Starting " + Arrays.toString(services));
 
         List<StartHandle> handlers = prepareStart(services, args);
-        if (hasSystemFlag("fakeStart")) return;
 
+        if (!hasSystemFlag("fakeStart")) {
+            doStart(handlers);
+        }
+
+        stream(services).forEach(this::executePostStartAction);
```

Comment:
post actions for a service shoud be executed after the service start. not after all services' startup. the logic must be invoked after the await() method

Reply:
fixed in 87dc88c18de9f94ba06edd58f7dd4e3112aa5a38